generator/sbpg/targets/rust.py

```diff
@@ -100,7 +116,27 @@ def commentify(environment: Environment,
   """
   Builds a comment.
   """
-  return indented_wordwrap(environment, value, indent=(" " * indent) + prefix, first=False, markdown=True)
+  value = indented_wordwrap(environment, value, indent=(" " * indent) + prefix, first=False, markdown=True)
+  value = wrap_urls(value)
+  value = escape_braces(value)
+  return value
+
+def wrap_urls(s):
+  urls = re.findall(r"\(?https?://[^\s\)]+\)?", s)
+  for url in urls:
+    if url.startswith("(") and url.endswith(")"):
+      continue
+    if url.endswith(")"):
+      s = s.replace(url, "<" + url[:-1] + ">)")
+    else:
+      s = s.replace(url, "<" + url + ">")
+  return s
+
+def escape_braces(s):
```

Comment:
Maybe for a future addition, it would be neat to see if we could somehow get python docstrings embedded into the rust generated docs seeing as how these are related to the crate generation.

Reply:
Yeah kind of like how the top of each message file tells you which yaml file it came from